client/src/lib/components/Icons/AnimatedBell/AnimatedBell.tsx

```diff
@@ -0,0 +1,371 @@
+        s: {a: 0, k: [100, 100, 100], ix: 6, l: 2},
+      },
+      ao: 0,
+      shapes: [
+        {
+          ty: 'gr',
+          it: [
+            {
+              ind: 0,
+              ty: 'sh',
+              ix: 1,
+              ks: {
+                a: 0,
+                k: {
+                  i: [
+                    [0, 0],
+                    [1.2, 12.4],
+                    [22.8, 8.2],
+                    [13.1, 0],
+                    [3.3, -11.7],
+                    [1.3, -33.1],
+                    [6.3, -6.1],
+                    [0, -9.5],
+                    [-9.2, 0],
+                    [0, 0],
+                    [0, 7.6],
+                    [8, 7.7],
+                  ],
+                  o: [
+                    [-6.2, -6.1],
+                    [-1.2, -33.1],
+                    [-3.4, -11.7],
+                    [-13.1, 0],
+                    [-22.9, 8.3],
+                    [-1.1, 12.4],
+                    [-8, 7.6],
+                    [0, 7.6],
+                    [0, 0],
+                    [9.1, 0],
+                    [0, -9.4],
+                    [0, 0],
+                  ],
+                  v: [
+                    [68.95, 39.5],
+                    [60.05, 8.9],
+                    [26.25, -56.6],
+                    [-0.05, -77.2],
+                    [-26.25, -56.6],
+                    [-60.15, 8.9],
+                    [-69.05, 39.5],
+                    [-85.55, 64.6],
+                    [-70.75, 77.2],
+                    [70.75, 77.2],
+                    [85.55, 64.6],
+                    [69.05, 39.5],
+                  ],
+                  c: true,
+                },
+                ix: 2,
+              },
+              nm: 'Path 1',
+              mn: 'ADBE Vector Shape - Group',
+              hd: false,
+            },
+            {
+              ty: 'fl',
+              c: {a: 0, k: [1, 1, 1, 1], ix: 4},
+              o: {a: 0, k: 100, ix: 5},
+              r: 1,
+              bm: 0,
+              nm: 'Fill 1',
+              mn: 'ADBE Vector Graphic - Fill',
+              hd: false,
+            },
+            {
+              ty: 'tr',
+              p: {a: 0, k: [0, 0], ix: 2},
+              a: {a: 0, k: [0, 0], ix: 1},
+              s: {a: 0, k: [100, 100], ix: 3},
+              r: {a: 0, k: 0, ix: 6},
+              o: {a: 0, k: 100, ix: 7},
+              sk: {a: 0, k: 0, ix: 4},
+              sa: {a: 0, k: 0, ix: 5},
+              nm: 'Transform',
+            },
+          ],
+          nm: 'Group 1',
+          np: 2,
+          cix: 2,
+          bm: 0,
+          ix: 1,
+          mn: 'ADBE Vector Group',
+          hd: false,
+        },
+      ],
+      ip: 0,
+      op: 576,
+      st: 0,
+      bm: 0,
+    },
+  ],
+  markers: [],
+};
+
+type AnimatedBellProps = Omit<AnimatedIconProps, 'source'>;
+
+export const AnimatedBell: React.FC<AnimatedBellProps> = forwardRef<
```

Comment:
Can we name this `BellIconAnimated` instead? So we align it with `BellIcon` and `LogoIconAnimated`. Also easier to find them in UI lib since they are sorted alphabetically there :) 
![image](https://github.com/29ki/29k/assets/474066/34dc55e1-2702-463b-9b20-721608743c4a)